src/index.js

```diff
@@ -10,7 +11,13 @@ makeServer();
 ReactDOM.render(
 	<React.StrictMode>
 		<Router>
-			<App />
+			<RegisterProvider>
+				<LoginProvider>
+					<AuthProvider>
+						<App />
+					</AuthProvider>
+				</LoginProvider>
+			</RegisterProvider>
```

Comment:
You can also create a main provider component and import all the providers there. This way you will have to import only one provider and your index.js will look much cleaner.